### DISCUSSION TITLE
Unshipped report - GET_FLAT_FILE_ACTIONABLE_ORDER_DATA

### DISCUSSION THREAD

**Discussion by olsw** [saleweaver/python-amazon-sp-api#115] [FAQ]
Is the report GET_FLAT_FILE_ACTIONABLE_ORDER_DATA_SHIPPING the same as the one report in the AWS docs which is GET_FLAT_FILE_ACTIONABLE_ORDER_DATA?
Only the former seems to work; if I try the latter I get a 403 error
However the former doesn't supply the same information as I get from the unshipped order reports from the site directly so very confused!
Thanks

**Comment by saleweaver** [comment]
You need to have PII access to get address data through the API, and you need to have assigned the correct roles to your application. 

Thanks,
Michael


On 4 Apr 2021 at 23:32 +0200, olsw ***@***.***>, wrote:


Is the report GET_FLAT_FILE_ACTIONABLE_ORDER_DATA_SHIPPING the same as the one report in the AWS docs which is GET_FLAT_FILE_ACTIONABLE_ORDER_DATA?
Only the former seems to work; if I try the latter I get a 403 error
However the former doesn't supply the same information as I get from the unshipped order reports from the site directly so very confused!
Thanks

**Reply by olsw** [reply]
Ah I should have put more information

The GET_FLAT_FILE_ACTIONABLE_ORDER_DATA_SHIPPING report works and includes address, what it doesn't include is the customisable URL which is what I really want to be able to access. Do you know if a) it would return at all if the role was wrong or if it would just redact some data and b) if the customisable URL would need specific PII access or role?
